.github/workflows/Tests.yaml

```diff
@@ -154,6 +161,38 @@ jobs:
           # The `path` is relative to the $GITHUB_WORKSPACE on the host machine
           path: docs-html
 
+  # Deploy built documentation to `gh-pages` on pushes to `develop`.
+  doc_deploy:
+    name: Deploy documentation
+    needs: doc_check
+    if: github.event_name == 'push' && github.ref == 'refs/heads/develop'
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v1
+      - name: Download built documentation
+        uses: actions/download-artifact@v1
+        with:
+          name: docs-html
+      - name: Add custom domain
+        if: github.repository == 'sxs-collaboration/spectre'
+        working-directory: ./docs-html
+        run: |
+          echo "spectre-code-.org" > CNAME
```

Comment:
only one `-` in `spectre-code.org`

Reply:
Copied this from https://github.com/sxs-collaboration/spectre/commit/5e933a48d6e063cac87f2f285a9cb2696208e8bd which has the additional `-`. Do you know why?

Reply:
Yea, I do know why... Something in the settings for the repo was wrong. That's fixed now.

Reply:
Was it wrong on Travis?

I fixed and squashed this.

Reply:
Was it wrong on Travis?

I fixed and squashed this.

Reply:
It was never used with Travis. That was something GitHub added recently while Larry and I were looking at changing where the documentation is hosted from.